src/notebookMiddlewareAddon.ts

```diff
@@ -173,7 +173,9 @@ export class NotebookMiddlewareAddon implements Middleware, Disposable {
             if (isOpen) {
                 // Send this to our converter and then the change notification to the server
                 const params = this.converter.handleRefresh(notebook);
-                client.sendNotification(DidChangeTextDocumentNotification.type, params);
+                if (params) {
```

Comment:
This was causing IW intellisense to be completely messed up (throws an exception in pylance)